src/main/java/uk/ac/ebi/eva/pipeline/io/writers/VepProcess.java

```diff
@@ -118,21 +137,28 @@ private void captureOutput(Process process, String vepOutput) {
     }
 
     public void write(byte[] bytes) throws IOException {
-        if (process == null) {
+        if (!isOpen()) {
             throw new IllegalStateException("Process must be initialized (hint: call open() before write())");
         }
         processStandardInput.write(bytes);
     }
 
+    public boolean isOpen() {
+        return process != null;
```

Comment:
not sure about adding this. if it died suddenly, trying to use it would result in an exception raised, which is what we want. 

On top of that, mixing `process != null` and `Process.isAlive()` would mix concerns on API usage and runtime validity. Using that hypothetical method would be also verbose because of that mixing